monai/inferers/inferer.py

```diff
@@ -64,6 +67,190 @@ def __call__(self, inputs: torch.Tensor, network: Callable, *args: Any, **kwargs
         raise NotImplementedError(f"Subclass {self.__class__.__name__} must implement this method.")
 
 
+class PatchInferer(Inferer):
+    """
+    Inference on patches instead of the whole image based on Splitter and Merger.
+    This splits the input image into patches and then merge the resulted patches.
+
+    Args:
+        splitter: a `Splitter` object that split the inputs into patches. Defaults to None.
```

Comment:
```suggestion
        splitter: a `Splitter` object that splits the inputs into patches. Defaults to None in which case inputs will be traversed in the batch dimension.
```
We should explain what the default values imply here, hopefully I have it right. Could this argument also just be typed as a callable?

Reply:
Thanks, that's correct! As the only interface of `Splitter` is `__call__`, it can be any Callable that act on the input. The input verification already checks for it only to be callable.

Reply:
I updated the checks. The splitter should be `Splitter` but since `Splitter` is basically a callable that get tensor and returns pairs of (patch, locations), any other callable that have the same behavior can be used (but not any callable). So to be more clear about it, I kept `Splitter` as the type hint but included `Callable` too. 
```python
splitter: Splitter | Callable | None = None,
```